vanilla.transform.cjs

```diff
@@ -0,0 +1,9 @@
+/***
+ * @vanilla-extract/jest-transform incorrectly maps its default export to "default", and so must be transformed itself
```

Comment:
nit: I don't understand how this comment maps to the underlying code, what is that we do to "map its default export to default" here?